WooCommerce/Classes/ViewModels/ManualTrackingViewModel.swift

```diff
@@ -237,7 +234,7 @@ final class AddCustomTrackingViewModel: ManualTrackingViewModel {
             trackingNumber?.isEmpty == false
 
         if returnValue {
-            saveSelectedCustomShipmentProvider()
+            saveSelectedShipmentProvider()
```

Comment:
I know it's pre-existing, but this still seems like a side-effect, as it was in `didSet`. This is a property which returns a bool on whether we can commit the change.... From the name alone, I would expect to be able to call it repeatedly without that triggering a save.